include/irr/asset/filters/CMatchedSizeInOutImageFilterCommon.h

```diff
@@ -100,7 +100,7 @@ class CMatchedSizeInOutImageFilterCommon : public CBasicImageFilterCommon
 			const core::SRange<const IImage::SBufferCopy> inRegions;
 			const core::SRange<const IImage::SBufferCopy> outRegions;
 			const IImage::SBufferCopy* oit;									//!< oit is a current handled region by executePerRegion lambda. Notice that the lambda may execute itself a few times with different oits since regions may overlap in a certain mipmap in an image!
-			core::vectorSIMDu32 offsetDifference, outByteStrides;
+			core::vectorSIMDi32 offsetDifference, outByteStrides; // @devshgraphicsprogramming @Criss I dont think it should ever get < 0, but there are cases where it tries to
```

Comment:
leave as is, see the comment on line `140`

Reply:
corrected in following commit